start/src/test/java/org/apache/accumulo/start/util/AsyncSocketAppenderTest.java

```diff
@@ -36,7 +31,7 @@
 
   @Before
   public void setUp() throws Exception {
-    sa = createMock(SocketAppender.class);
+    sa = EasyMock.createMock(SocketAppender.class);
```

Comment:
My IDE was having problems with the static imports.  Not sure why.  I could have looked into it more but I am not a fan of static imports of methods like that as it makes the code harder to read.  Therefore, I removed them.